packages/frontend/app/components/user-profile-calendar.js

```diff
@@ -1,34 +1,49 @@
 import Component from '@glimmer/component';
 import { service } from '@ember/service';
 import { DateTime } from 'luxon';
-import { tracked } from '@glimmer/tracking';
+import { tracked, cached } from '@glimmer/tracking';
 import { action } from '@ember/object';
-import { dropTask } from 'ember-concurrency';
 import { sortBy } from 'ilios-common/utils/array-helpers';
+import { TrackedAsyncData } from 'ember-async-data';
 
 export default class UserProfileCalendar extends Component {
   @service fetch;
   @service iliosConfig;
   @service userEvents;
+  @service localeDays;
+
   @tracked date = new Date();
-  @tracked calendarEvents;
 
-  load = dropTask(async () => {
-    //luxon weeks always stort on Monday to we have to adjust for that
```

Comment:
"stort" :laughing:

Reply:
Didn't really need to git blame to know who that line belonged to 😁